docs/admin/extensible-admission-controllers.md

```diff
@@ -0,0 +1,152 @@
+---
+assignees:
+- smarterclayton
+- lavalamp
+- whitlockjc
+- caesrxuchao
+title: Extending admission controllers
+---
+
+* TOC
+{:toc}
+
+## The plugin-style admission controllers are not flexible enough
+
+[Using Admission Controllers documentation](/doc/admin/admission-controllers.md) introduces how to use the plugin-style admission controllers.The plugin admission controllers need to be compiled in-tree, and are only configurable when the apiserver starts up.
+
+## How do I extend admission controllers?
```

Comment:
*Style change*: Titles for sections should be phrases, not questions.

For example, instead of "How do I extend admission controllers?", this section should be called "Extending Admission Controllers"